.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,6 +5,10 @@ Include "Fixes #nnn" if you are fixing an existing issue.
 In "Component Name" section write which component is changed in this PR. This
 will help us review your PR quicker.
 
+In "Test Plan" please detail exactly how you plan to test this PR with enough
```

Comment:
In "Test Plan," provide enough detail on how you plan to test this PR so that a reviewer can validate your tests. If our CI covers sufficient tests, then state which tests cover the change.

Reply:
I _think_ I said this at the top? I can duplicate it here too if you like :)

Reply:
A rule of thumb for technical writing is not use "please" so I rewrote those sentences.

Reply:
Thanks :) I personally prefer to use polite language; but I'm oaky with this! I'm not a Technical Writer!

Reply:
> A rule of thumb for technical writing is not use "please"

```bash
184 ilyam:netdata (master)$ grep -i "please" * -R | wc -l
     201
```


And what about `kindly` @shortpatti ? 😆 

> packaging/installer/methods/offline.md:# For binaries for OS types and architectures not listed on [go.d releases](https://github.com/netdata/go.d.plugin/releases/latest), kindly open a github issue and we will do our best to serve your request

Reply:
@ilyam8 Stop trol'n 🤣